source/specify-documents-to-return.txt

```diff
@@ -0,0 +1,179 @@
+direction, the method defaults to sorting in ascending order.
+
+The following example returns all documents with the ``cuisine`` value of
+``"Italian"``, sorted in ascending order:
+
+.. io-code-block::
+
+   .. input:: /includes/specify-documents-to-return.java
+      :start-after: start-sort-method
+      :end-before: end-sort-method
+      :language: java
+
+   .. output::
+
+      (Lewis Drug Store) Locanda Vini E Olii
+      101 Restaurant And Bar
+      44 Sw Ristorante & Bar
+      900 Park
+      A Voce
+      ...
+      Zucchero E Pomodori
+
+Skip
+----
+
+To skip a specified number of documents before returning your query results,
+chain the ``skip()`` method to a ``find()`` method and pass the number of documents to skip. The
+``skip()`` method ignores the specified number of documents in your query
+results and returns the rest.
+
+The following example returns all documents that have a ``borough`` field value
+of ``"Manhattan"``, and skips the first 10 documents:
+
+.. io-code-block::
+
+   .. input:: /includes/specify-documents-to-return.java
+      :start-after: start-skip
+      :end-before: end-skip
+      :language: java
+
+   .. output::
+
+      Cafe Metro
+      Lexler Deli
+      Domino'S Pizza
+      ...
+
+Combine Limit, Sort, and Skip
+-----------------------------
+
+You can combine the ``limit()``, ``sort()``, and ``skip()`` methods in a single
+operation by chaining them together. This allows you to set a maximum number of sorted documents to
+return, skipping a specified number of documents before returning.
+
+The following example returns ``5`` documents that have a ``cuisine`` value of
+``"Italian"``. The results are sorted in ascending order by ``name`` field value,
+skipping the first ``10`` documents.
+
+.. io-code-block::
+
+   .. input:: /includes/specify-documents-to-return.java
+      :start-after: start-limit-sort-skip
+      :end-before: end-limit-sort-skip
+      :language: java
+
+   .. output::
+      
+      Acqua
+      Acqua Restaurant
+      Acqua Santa
+      Acquista Trattoria
+      Acquolina Catering
+
+.. note::
+
+   The order in which you call these methods doesn't change the documents
+   that are returned. The driver automatically reorders the calls to perform the
+   sort and skip operations first, and the limit operation afterward.
+
+Additional Information
+----------------------
+
+.. TODO: uncomment when merged in.
+.. For more information about specifying a query, see :ref:`java-rs-specify-query`.
+
+.. For more information about retrieving documents, see :ref:`java-rs-retrieve`.
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+To learn more about any of the methods or types discussed in this
+guide, see the following API Documentation:
+
+- `find() <{+api+}/mongodb-driver-reactivestreams/com/mongodb/reactivestreams/client/MongoCollection.html#find()>`__
+- `FindPublisher <{+api+}/mongodb-driver-reactivestreams/com/mongodb/reactivestreams/client/FindPublisher.html>`__
```

Comment:
[nit] Can potentially move this to the beginning or end of the list so all methods are grouped together.